examples/dnn_instance_segmentation_ex.cpp

```diff
@@ -16,7 +16,7 @@
        ./dnn_instance_segmentation_ex /path/to/VOC2012-or-other-images
 
     An alternative to steps 2-4 above is to download a pre-trained network
-    from here: http://dlib.net/files/instance_segmentation_voc2012net.dnn
+    from here: http://dlib.net/files/instance_segmentation_voc2012net_v2.dnn
```

Comment:
For the time being at least, this file is available here:
https://www.dropbox.com/s/t7gi4lfw6mrx10v/instance_segmentation_voc2012net_v2.dnn